src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+$square-size: 100px;
+$square-color: #eee;
+$square-color-hover: #ffbfcb;
+$padding: 10px;
+$border-base: 1px solid black;
+$font-family: Arial, Helvetica, sans-serif;
+$font-size: 30px;
```

Comment:
u don't need variables for these properties. Remove it.
1) In the future - we will analyze the option of how to add variables for fonts
2) in the case of a border - u can create a variable for color and no more
3) padding - u used inly once